### DISCUSSION TITLE
Toggle checkboxes in issues/PRs

### DISCUSSION THREAD

**Discussion by wopian** [community/community#4508] [Mobile]
On the web app, maintainers can toggle the status of checkboxes on the rendered markdown of issues and pull requests. 

This is used by many bots, such as Renovate - where checking a box instructs the bot to rebase the PR it opened. Right now I have to either edit the PR description manually or open the PR in the browser to accomplish this.

**Comment by oscarb** [comment]
Agree! Would be super nice if the mobile app could be utilized better for checklists ✅
